config/config.exs

```diff
@@ -17,3 +17,12 @@ config :monarch, Oban,
   name: Monarch.Oban,
   repo: Monarch.Repo,
   testing: :manual
+
+case Mix.env() do
+  :test ->
+    import_config "test.exs"
+
+  _ ->
+    # No action for other environments
+    nil
+end
```

Comment:
The default from a new mix project is probably okay here. Let it fail™ if there's no file for the env

```suggestion
import_config "#{config_env()}.exs"
```